wordpress-mobile/WordPress-iOS
# Update Aztec-Editor to 1.12 and add block editor paragraph alignment

This updates the AztecEditor to [version 1.12](https://github.com/wordpress-mobile/AztecEditor-iOS/releases/tag/1.12.0) and brings in the changes for paragraph alignment that needed that update.

To test:

* Smoke test of Aztec Editor
* Check paragraph alignment behavior in gutenberg editor (previously [this issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1545) was observed and filed, but I'm actually not seeing that issue anymore, cc: @SergioEstevao )
* Check background color appearance in quote blocks

### TODO

- [ ] Update gutenberg ref after https://github.com/wordpress-mobile/gutenberg-mobile/pull/1489 is merged

### Update release notes
